templates/security/fips.html

```diff
@@ -213,15 +218,15 @@ <h2>Certified packages under FIPS 140</h2>
       </tbody>
     </table>
     <a class="p-button--neutral js-invoke-modal" href="/security/certifications/docs/fips">Read more about the FIPS certification</a>
-    <a class="p-button--positive js-invoke-modal" href="/support/contact-us">Access the FIPS140 validated modules</a>
+    <a class="p-button--positive js-invoke-modal" href="/advantage">Access the FIPS140 validated modules</a>
   </div>
 </section>
 
 <section class="p-strip">
   <div class="u-fixed-width">
     <h2>Canonical patches vulnerabilities</h2>
     <p>Each FIPS 140 certificate is valid for 5 years, however vulnerabilities happen, and it is our intention to publish fixed packages quickly, irrespective of their certification status. We therefore provide two alternative options. An option to remain with the certified cryptographic packages (called the 'fips' option), and an option to use the certified packages but include security fixes (called the 'fips-updates' option) when available. Check <a class="p-link--external" href="https://security-certs.docs.ubuntu.com/en/fips">our security pages on how to enable these options</a>.</p>
-    <p>We recommend enabling the 'fips-updates' option that includes the security fixes. The packages from 'fips-updates' option are updated to include high and critical security fixes during the whole product lifecycle including the <a href="/security/esm">Extended Security Maintenance phase</a>.</p>
+    <p>We recommend enabling the 'fips-updates' option that includes the security fixes. The packages from 'fips-updates' option are updated to include high and critical security fixes during the whole product lifecycle including the <a href="/security/esm">Extended Security Maintenance (ESM) phase</a>.</p>
```

Comment:
The link in the copy docs finishes at "Maintenance" can you close the `</a>` before (ESM)? Thanks

Reply:
Thanks, I've changed it and updated the copydoc as well to be identical.